packages/platform-test-suite/bin/test.sh

```diff
@@ -201,7 +209,7 @@ then
   cmd="${cmd} NODE_ENV=test node_modules/.bin/mocha -b ${scope_dirs}"
 else
   echo $cmd
-  cmd="${cmd} NODE_ENV=test yarn mocha --inspect-brk -b ${scope_dirs}"
```

Comment:
Good catch
It definitely blocks the ci

Reply:
Actually it shouldn't block CI, because on line 199 we check if we are in the CI environment or not. But regardless, I don't like to have it run in debug mode by default.